compose-stylesheet-core/src/commonMain/kotlin/com/moriatsushi/compose/stylesheet/content/ProvideContentStyle.kt

```diff
@@ -1,4 +1,4 @@
-package com.moriatsushi.compose.stylesheet
+package com.moriatsushi.compose.stylesheet.content
 
```

Comment:
hey there, please remove the unnecessary space

Reply:
Sorry, where is the space you mentioned?

Reply:
sorry, i apologized for the following